Aspire Budgeting/Facilities/GoogleSheetsManager.swift

```diff
@@ -133,4 +186,86 @@ final class GoogleSheetsManager: ObservableObject {
       }
     }
   }
+  
+  private func createSheetsValueRangeFrom(amount: String,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/a77c2477cb8c75fe2389ac27f223c44a1a0476c6/Aspire Budgeting/Facilities/GoogleSheetsManager.swift#L190"></span>Function should have 5 parameters or less: it currently has 6<br />
<code>function_parameter_count</code> <code>GoogleSheetsManager.swift:190</code></td>
    </tr>
  </tbody>
</table>